### ISSUE TITLE
Accessibility: Color-Contrast too low on package page

### ISSUE THREAD

**Issue by robertmuehsig** [NuGet/NuGetGallery#7847]
### Describe the bug
I played around with [Accessibility Insights](https://accessibilityinsights.io/) and tried it on NuGet.org. The site seems (at least from the "Fast pass") quite good regarding accessibility, but the color on the package page for the currently released package is too low.

### To Reproduce
1. On this URL 'https://www.nuget.org/packages/Newtonsoft.Json/'
2. Perform this operation 'FastPass with https://accessibilityinsights.io/'
3. See error

### Expected behavior
"Congratulations!
No failed automated checks were found."

### Screenshots
![image](https://user-images.githubusercontent.com/756703/74595115-19d7f800-503e-11ea-8730-fb319a456680.png)

### Additional context
The link color and the background color (this blue-ish color) are problematic. I'm not sure if this combination is used on other pages as well. I would guess that the best fix would be to change the background color of the "bg-info" class. 

I could try to find a color combination, but I guess there is not "right" or "wrong", but if you would like that I should try something just give me a comment here. :)

**Comment by skofman1** [comment]
Thanks for reporting! Adding this work item to the WCAG 2.1 epic.

**Comment by ryuyu** [comment]
I believe this is now fixed!
Thanks for the report!
